drivers/tee/tee_core.c

```diff
@@ -111,6 +121,209 @@ static int tee_release(struct inode *inode, struct file *filp)
 	return 0;
 }
 
+static const char *tee_session_get_application_id(void)
```

Comment:
These two functions seems rather useless. Until they do anything useful how about just skipping them?

Reply:
idea is that in here we would have in example SMACK or SELnux labels for particular application. Eg. get the application label from the context. but that code I do not have at the moment.

I believe this should be some kernel method to configure source for application id and then per system this is selected. Eg. if SMACK is in use then smack application label from current process would be used. Some may have non-upstreamed ways to deliver application ID and then they could provide with this mechanism.

Idea was here to illustrate that something like this is needed.

Reply:
> Idea was here to illustrate that something like this is needed.

OK, that's good for discussion. But if it's to go into the kernel we either need something specific and usable or have it removed.

Reply:
I was also thinking an option to split this commit as two.

One which does user and group and then leaves application part to own commit as RFC.